Source/MediaInfo/Audio/File_Usac.cpp

```diff
@@ -334,6 +673,96 @@ File_Usac::~File_Usac()
 {
 }
 
+//***********************************************************************
+// Temp
+//***********************************************************************
+
+//---------------------------------------------------------------------------
+void File_Usac::hcod_sf(const char* Name)
+{
+    Element_Begin1(Name);
+    int16u Pos=0;
+    int32u index=0;
+
+    for (;;)
+    {
+        int8u h;
+        Peek_S1(2, h);
+        index = huffman_scl[index][h]; /* Expensive memory access */
+        if (index & 1)
+        {
+            if (index & 2)
+                Skip_SB(                                        "huffman");
+            else
+                Skip_S1(2,                                      "huffman");
+            break;
+        }
+        Skip_S1(2,                                              "huffman");
+        index >>= 2;
+
+        if (Pos>240)
+        {
+            Skip_BS(Data_BS_Remain(),                           "Error");
+            Element_End0();
+            return;
+        }
+    }
+    Element_Info1(huffman_scl[Pos][0]-60);
+    Element_End0();
+}
+
+//----------------------------------------------------------------------------
+int32u File_Usac::arith_decode(int16u& low, int16u& high, int16u& value, const int16u* cf, int32u cfl)
+{
+    int32u range=(int32u)(high-low+1);
+    int32u cm=((((int)(value-low+1))<<14)-((int)1))/range;
+    const int16u* p=cf-1;
+
+     do
+     {
+         const int16u* q=p+(cfl>>1);
+         if (*q>cm)
+         {
+             p=q;
+             cfl++;
+         }
+         cfl>>=1;
+     }
+     while (cfl>1);
+
+    int32u symbol=(int32u)(p-cf+1);
+    if (symbol)
+        high=low+((range*cf[symbol-1])>>14)-1;
+    low+=(range*cf[symbol])>>14;
+
+    for (;;)
+    {
+        if (high&0x8000)
+        {
+            if (!(low&0x8000))
+            {
+                 if (low&0x4000 && !(high&0x4000))
+                 {
+                     low-=0x4000;
+                     high-=0x4000;
+                     value-=0x4000;
+                 }
+                 else
+                     break;
+            }
+        }
+
+        low=low<<1;
+        high=(high<<1) | 1;
+
+        bool bit;
+        Get_SB(bit,                                 "arith_data");
+        value=(value<<1)|bit;

```

Comment:
"\MediaInfoLib\Source\MediaInfo\Audio\File_Usac.cpp(760,26): warning C4805: '|': unsafe mix of type 'int' and type 'bool' in operation" from MSVC.

"value=(value<<1)|(int)bit;" for avoiding the warning.